data/flexibleForms/caseStatus/caseStatus.ts

```diff
@@ -57,12 +57,22 @@ const form: Form = {
           question: 'Start Date',
           type: 'date',
           required: true,
-          conditions: [
-            {
-              id: 'type',
-              value: 'CIN',
-            },
-          ],
+          conditions: {
+            OR: [
```

Comment:
Didn't know you could use OR, good to know!

Reply:
You couldn't, I added this here: https://github.com/LBHackney-IT/lbh-social-care-frontend/pull/859/files#diff-9893684b9b99f4716e2adf2b28835c8e6f78037ecb1346a1e7a091712dc3bb75

Reply:
Yup, noticed that :) Will definitely use this approach next time we have a similar problem to solve :)